packages/js/settings-editor/src/tests/route.test.tsx

```diff
@@ -0,0 +1,200 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/element';
+import { renderHook } from '@testing-library/react-hooks';
+import { screen, render } from '@testing-library/react';
+import { addAction, applyFilters, didFilter } from '@wordpress/hooks';
+/* eslint-disable @woocommerce/dependency-group */
+// @ts-ignore No types for this exist yet.
+import { privateApis } from '@wordpress/router';
+/* eslint-enable @woocommerce/dependency-group */
+
+/**
+ * Internal dependencies
+ */
+import { useActiveRoute, useModernRoutes } from '../route';
+
+// Mock external dependencies
+jest.mock( '@woocommerce/navigation', () => ( {
```

Comment:
This can be removed now

Reply:
Removed, thanks. 15a3ca848d0f28c9dffc26a2374734db129003e1